latex/chapter/Konzeption.tex

```diff
@@ -167,3 +167,54 @@ \subsubsection{Software Architektur}
 
 % Beschreibung der Architektur
 % GGF Sequenzdiagramm
+
+\subsection{Demosystem}
+Das Demosystem baut auf dem Versuchssystem auf.
+Wie in den Anforderungen beschrieben, ist die Authentifizierung und damit auch das Versuchssystem Teil des Back-Ends.
+\begin{figure}
+    \centering
+    \includegraphics[width=0.8\textwidth, keepaspectratio]{images/Architektur-Demosystem.png}
+    \caption{Architektur Demosystem}
+    \label{fig:ArchitectureDemoSystem}
+\end{figure}
+In Abbildung~\ref{fig:ArchitectureDemoSystem} ist die grundlegende Architektur des Demosystems dargestellt.
+Diese unterteilt sich in die zwei Elemente: Client (Front-End) und Server (Back-End).
+
+\subsubsection{Server}
+Der Server erhält dafür einen eigenen Einstiegspunkt, welcher eine Schnittstelle für den Client bereitstellt.
+Gleichzeitig fungiert der Server als Authentifizierungssystem.
+Dafür werden die Hauptkomponenten des Versuchssystem (DataSetHandler, AudioPreprocessor und FeatureExtractor) vom Server implementiert.
+
+\begin{figure}
```

Comment:
Wir brauchen da ein einheitliches Format, in meinen Augen muss das Bild dahin wo mans braucht und nicht an den Start der Seite

Reply:
Mir ist das tatsächlich relativ egal. Ich vertrete die Meinung dass Latex das managen soll, aber ich bin fein damit alles an die "richtige" fixe position zu setzen -> Änder ich!

Reply:
find das für den Lesefluss absolut unpraktisch, fehlt ja nur das `[H]`